lib/core/src/server/build-dev.js

```diff
@@ -117,7 +117,11 @@ export async function buildDevStandalone(options) {
     if (options.smokeTest) {
       process.exit(stats.toJson().warnings.length ? 1 : 0);
     } else if (!options.ci) {
-      opn(address);
+      opn(address).catch(() => {
+        logger.error(`Could not open ${address} inside a browser. If you're running this command inside a
+docker container or on a CI, you need to pass the '--ci' flag to prevent opening a
```

Comment:
You can use `common-tags` and keep the whitespace easier to read.

Not a blocker for merger to me.

Reply:
I'll add common-tags